gravitee-apim-portal-webui/package.json

```
@@ -33,7 +33,7 @@
     "@angular/platform-browser-dynamic": "17.0.5",
     "@angular/router": "17.0.5",
     "@asciidoctor/core": "3.0.2",
-    "@asyncapi/web-component": "1.0.0-next.47",
+    "@asyncapi/web-component": "^1.2.23",
```

```suggestion
    "@asyncapi/web-component": "1.2.23",
```

We should pin the version otherwise it will open a renovate PR 🙈